scheduler/client/cli/cli.go

```diff
@@ -5,23 +5,16 @@ import (
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	commoncli "github.com/twitter/scoot/common/client"
 	"github.com/twitter/scoot/common/dialer"
+	smoketest "github.com/twitter/scoot/integration-tests/smoketest"
 	"github.com/twitter/scoot/scheduler/api/thrift/gen-go/scoot"
 	"github.com/twitter/scoot/scheduler/client"
 )
 
 // Scoot API Client interface that includes CLI handling
-type CLIClient interface {
-	Exec() error
-}
-
-// Implements CLIClient - basic
-type simpleCLIClient struct {
-	rootCmd     *cobra.Command
-	addr        string
-	dial        dialer.Dialer
-	logLevel    string
-	scootClient *client.CloudScootClient
+type SimpleCLIClient struct {
+	simpleCLIClient commoncli.SimpleClient
```

Comment:
we could embed the commoncli.SimpleClient here as an unnamed field. Means we can skip a layer of nested calls (e.g. c.simpleCLIClient.RootCmd.Execute() becomes c.RootCmd.Execute())

Reply:
We can embed it as an unnamed field but the struct is being passed to the cmd.Run() function, I'm unsure if we can pass it if it is unnamed.

Reply:
Thanks for pointing this out, done.